apis/kueue/v1beta1/provisioningrequestconfig_types.go

```diff
@@ -21,10 +21,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type MergePodSetsMode string
```

Comment:
```suggestion
type ProvisioningRequestConfigMergePodSetsMode string
```

Could you add a prefix to avoid the conflicts between the API directory?